### ISSUE TITLE
Feature(CLI): Make CLI output more CI friendly 

### ISSUE THREAD

**Issue by erezrokah** [cloudquery/cloudquery#2554]
### Is your feature request related to a problem? Please describe.

Noticed this while updating our GitHub action. Our CLI output is a bit hard to follow when running in CLI, for example:
https://github.com/cloudquery/setup-cloudquery/actions/runs/3213876151/jobs/5253865421#step:7:23

Our progress bar floods stdout in non interactive environments.

### Describe the solution you'd like

Better output in non interactive environments (maybe less frequent updates of the console).

### Describe alternatives you've considered

Keeping this as is

### Additional context

I'd say low priority issue, but maybe there's an easy fix for it

**Comment by yevgenypats** [comment]
I think this can turn out to a never ending dev work implementing two UI modes: UI mode for interactive and UI mode for non-interactive environment as it can even go further to third mode (depending on the environment).

When running in CI I think it is up to the user to switch to `--enable-console-log`. because otherwise amount of flags will explode quickly like - `--frequency-progress-bar` (that depends on the environment). `--frequency-download-progress-bar`, `--quiet-download-progress-bar` and so on for every print that we have. 

for example wget just have `quiet` mode - https://www.baeldung.com/linux/wget-show-only-progress-bar#:~:text=The%20%2Dq%20option%20turns%20off,available%20since%20GNU%20wget%201.16. 

we can have quiet mode but then it will be pretty much the same as `--enable-console-log` but potentially less useful so I wonder if really need it.

**Comment by erezrokah** [comment]
> I think this can turn out to a never ending dev work implementing two UI modes: UI mode for interactive and UI mode for non-interactive environment as it can even go further to third mode (depending on the environment).
> 
> When running in CI I think it is up to the user to switch to `--enable-console-log`. because otherwise amount of flags will explode quickly like - `--frequency-progress-bar` (that depends on the environment). `--frequency-download-progress-bar`, `--quiet-download-progress-bar` and so on for every print that we have.
> 
> for example wget just have `quiet` mode - https://www.baeldung.com/linux/wget-show-only-progress-bar#:~:text=The%20%2Dq%20option%20turns%20off,available%20since%20GNU%20wget%201.16.
> 
> we can have quiet mode but then it will be pretty much the same as `--enable-console-log` but potentially less useful so I wonder if really need it.

💯 Agree, I'd expect the progress bar library to handle it via an option or something and we shouldn't implement this on our own.

What `--enable-console-log` does and how it changes the progress bar?

**Comment by yevgenypats** [comment]
`--enable-console-log` just closes stdout so all `fmt.Println` are silent and logs are by default going into `stderr` so you will see only those.

**Comment by yevgenypats** [comment]
Closing this (trying to make some cleanups of issues for next week planning) as I think this should be good enough for now (please reopen if needs more discussion).
